readabilipy/simple_json.py

```diff
@@ -52,8 +52,13 @@ def simple_json_from_html_string(html, content_digests=False, node_indexes=False
         # Call Mozilla's Readability.js Readability.parse() function via node, writing output to a temporary file
         article_json_path = f_html.name + ".json"
         jsdir = os.path.join(os.path.dirname(__file__), 'javascript')
-        subprocess.check_call(
-            ["node", "ExtractArticle.js", "-i", html_path, "-o", article_json_path], cwd=jsdir)
+        subprocess.run(
+            ["node", "ExtractArticle.js", "-i", html_path, "-o", article_json_path],
+            check=True,
+            cwd=jsdir,
+            stdout=subprocess.DEVNULL,
+            stderr=subprocess.DEVNULL,
+        )
```

Comment:
Instead of unilaterally swallowing output, how about redirecting it and only showing it if there's a subprocess execution error?

```suggestion
        try:
             result = subprocess.run(
                 ["node", "ExtractArticle.js", "-i", html_path, "-o", json_path],
                 cwd=jsdir,
                 check=True,
                 stdout=subprocess.PIPE,
                 stderr=subprocess.PIPE,
                 universal_newlines=True)
         except subprocess.CalledProcessError as e:
             print(e.stderr)
             raise
```